ecc_enclave/enclave/enclave.cpp

```diff
@@ -99,13 +114,18 @@ int ecall_cc_invoke(const uint8_t* signed_proposal_proto_bytes,
         // TODO: encrypt fabric Response object
         crm = {};
 
+        {  // encrypt response
+            encrypted_response = pdo::crypto::skenc::EncryptMessage(return_encryption_key,
+                ByteArray(b64_response.c_str(), b64_response.c_str() + b64_response.length()));
+        }
+
```

Comment:
You have an `encrypt_message` in cc-data which would be nicer to use here?  Also would make error checking consistent with above? (right now an error would cause an exception which i'm not sure is even caught be us?)

Reply:
good catch, that function was meant exactly for that